packages/view-util/lib/filters.js

```diff
@@ -0,0 +1,43 @@
+import { view, viewBuilder } from '@view-builder/core/ns.js'
+import { rdf, schema } from '@tpluscode/rdf-ns-builders'
+
+export function generateLookupSources(filter) {
+  filter
+    .deleteOut(view.operation)
+    .addOut(view.operation, view.Eq)
+
+  filter
```

Comment:
Sometimes these things are lists.

See: https://github.com/zazuko/rdf-cube-view-query/blob/ed0cdf45dabf0da0373d85583d6deb1e4dc7c46a/lib/Filter.js#L62

Reply:
Right. They are not going to be lists when created in view builder but no harm adding a `deleteList` :)